notebooks/official/sdk/sdk2_bigframes_tensorflow.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>from trained model. -&gt; from the trained model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2256/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>